packages/api/src/helpers/extensionMessaging.ts

```diff
@@ -92,8 +92,12 @@ const serializeMessage = (msg: APIMessages): any => {
     modifiedMsg.payload.amount = JSON.stringify(modifiedMsg.payload.amount);
   }
 
-  if (typeof modifiedMsg.payload?.limitAmount === 'object') {
```

Comment:
Was this removed on purpose?

Reply:
Actually no it's probably a rebase-thing